### PR TITLE
ui: Use vue-multiselect settings pages

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

**Review by MarionGiusti — APPROVED** [review]

**Review by pirlgon — APPROVED** [review]

**Comment by MarionGiusti** [comment]
Now, when we add a new user, all 3 groups are checked by default, is that what you wanted ?

**Comment by pgiraud** [comment]
Ready for review.

**Review by bersace — APPROVED** [review]

**Review by pirlgon — APPROVED** [review]

**Review by MarionGiusti — COMMENTED** [review]
Pas important mais il y a des typos ds le message de commit : 'it seemS' et 'old version of selEnium'.

File: tests/test_20_inventory.py [hunk 1]
```diff
@@ -53,8 +53,7 @@ def test_edit_instance(registered_agent, browser, ui_url):
     comment = browser.select("#inputCommentUpdate").get_attribute("value")
     assert "Registered by tests." == comment
     multiselect = MultiSelect(browser, "selectGroupsUpdate")
-    multiselect.toggle().unselect("default")
-    multiselect.toggle().select("default")
+    assert "default" in multiselect.selected[0].text
```
Comment (pgiraud) [inline tests/test_20_inventory.py:56]: Can you try?
```suggestion
    assert multiselect.selected[0].text == "default"
```

**Review by MarionGiusti — APPROVED** [review]

File: tests/test_20_inventory.py (hunk 1 above)
Comment (MarionGiusti) [reply tests/test_20_inventory.py:56]: yes sorry, I made it in pdb but keep "in" in my code.. I make the change

**Pull request merged**